### PR TITLE
Fix the CodeQL very slow issue

### PR DESCRIPTION
**Pull request by nigma456**
##### Objective
Recently it takes very long time(~50 minutes) to run the codeql on Github CI due to the test-resources folder
##### Abstractions
Add the test-resources folder to the configure file codeql-config.yml so that the codeql will not scan it.
##### Tests performed
Passed local test and Github CI

### CODE REVIEW

File: .github/codeql/codeql-config.yml [hunk 1]
```diff
@@ -7,4 +7,7 @@ paths:
 - '**/extension/src/**'
 
 paths-ignore:
-- '**/.vscode-test/**'
+  - '**/.vscode-test/**'
+  - '**/.vscode-test/**'
```
Comment (Sen-real) [inline .github/codeql/codeql-config.yml:11]: this is duplicate

File: .github/codeql/codeql-config.yml [hunk 2]
```diff
@@ -7,4 +7,7 @@ paths:
...
+  - '**/AutoLispExt/out/**/*.js'
```
Comment (Sen-real) [inline .github/codeql/codeql-config.yml:12]: I guess you meant **/AutolispExt/out/test/**

Comment (nigma456) [reply .github/codeql/codeql-config.yml:12]: No, we should not scan the compiled js file under out folder.

**Review by Sen-real — APPROVED** [review]

File: .github/codeql/codeql-config.yml (hunk 2 above)
Comment (Sen-real) [reply .github/codeql/codeql-config.yml]: I see.

**Pull request merged**